### ISSUE TITLE
Bug in ctc_ops.py 's documentation

### ISSUE THREAD

**Issue by argman** [tensorflow/tensorflow#11778]
see in 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/ctc_ops.py#L201
it says the inputs is logits, but from the op test https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/kernel_tests/ctc_decoder_ops_test.py#L111 
 we can see the input should be log probability .
Although seems this does not affect the decoder's output, but it affects the decoder's return of path's log_prob, and the discribe of neg_sum_logits is confusing.
I think the document should be changed, the inputs should be log of probabilities and the output should be -log(p), maybe we can transform the output to p with tf.exp(-p), i think probability output is more comfortable

and confusing people from another issue #6034

**Comment by aselle** [comment]
@ebrevdo, please take a look,as it seems you have touched this file quite a bit.

**Comment by ebrevdo** [comment]
The op does a normalization in log space, so logits are an acceptable input.  In the unit test you could shift the log probs by any constant value and the output and gradients would still be correct.
